runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -427,6 +432,22 @@ MM_CopyForwardScheme::clearGCStats(MM_EnvironmentVLHGC *env)
 	static_cast<MM_CycleStateVLHGC *>(env->_cycleState)->_vlhgcIncrementStats._continuationStats.clear();
 }
 
+#if defined(J9VM_GC_SPARSE_HEAP_ALLOCATION)
+void
+MM_CopyForwardScheme::recycleLeafRegionsForVirtualLargeObjectHeap(MM_EnvironmentVLHGC *env, uintptr_t arrayletLeafCount)
```

Comment:
I'd rename the method name to recycleReservedRegionsForVirtualLargeObjectHeap
and use reservedRegionCount instead of arrayletLeafCount